### PR TITLE
CAS-56 Feature/channel property in search

### PR DESCRIPTION
**Pull request by andriizhumela**
Added `Channel` field to search message result.

### CODE REVIEW

File: library/src/main/java/com/getstream/sdk/chat/rest/controller/RetrofitClient.java [hunk 1]
```diff
@@ -112,4 +114,10 @@ public static Retrofit getAuthorizedCDNClient(CachedTokenProvider tokenProvider,
                 .addConverterFactory(GsonConverterFactory.create(GsonConverter.Gson()))
                 .build();
     }
+
+    private static HttpLoggingInterceptor setupLoggingInterceptor() {
+        loggingInterceptor = new HttpLoggingInterceptor();
+        loggingInterceptor.level(HttpLoggingInterceptor.Level.BODY);
```
Comment (elevenetc) [inline library/src/main/java/com/getstream/sdk/chat/rest/controller/RetrofitClient.java]: @tbarbugli do you remember if we disabled it deliberately some time ago or not?

Comment (andriizhumela) [reply library/src/main/java/com/getstream/sdk/chat/rest/controller/RetrofitClient.java]: @elevenetc i can disable it if needed

**Comment by elevenetc** [comment]
@andriizhumela lets add also search messages into Sample app. Currently `client.searchMessages` is not even used and we need to test search eventually as it's a basic feature of almost any messaging app. 

`ChannelActivity/ChannelViewModel` is too complicated right now, so it might be hard to add search there. So I would add search icon in Toolbar of `ChannelActivity` and open new activity `MessageSearchActivity`. Where we can have edit text and recycler view for messages.

**Comment by andriizhumela** [comment]
@elevenetc Add to sample - Ok. But what i should to do with loggingInterceptor?

**Comment by andriizhumela** [comment]
@elevenetc Added sample with search.
Would be nice add metadata to search response for correct pagination abbillity.
Also would be nice add somthing like scrollToMessageById feature to messageList. For auto scroll to message what we looked.

**Comment by andriizhumela** [comment]
Added search to ChannelActivity.

**Pull request merged**